locale/templates/LC_MESSAGES/amo.pot

```diff
@@ -32,8 +32,8 @@ msgstr ""
 msgid "Added to %(collectionName)s"
 msgstr ""
 
-# This is a header for a list meaning Add to [some collection name]
 #: src/amo/components/AddAddonToCollection/index.js:269
+#. This is a header for a list meaning Add to [some collection name]
```

Comment:
What's with the `.` prefix ?

Reply:
I explained in the issue: https://github.com/mozilla/addons-frontend/issues/12756 - this is a "generated" comment, which is what Pontoon uses to display comments. Previously, we were generating "translator" comments.

Translator comments are left by localizers so that was incorrect. Generated comments are extracted from the source code.

Reply:
Oh, I compared the ones generated by addons-server but focused on the lack of `L10n:` string in that comment (which was discussed previously on Matrix, it's fine) but missed the fact that addons-server ones do have the leading `.`. Carry on.